openwisp_radius/tests/test_api.py

```diff
@@ -335,3 +336,22 @@ def test_user_auth_token_404(self):
                     password='tester')
         r = self.client.post(url, opts)
         self.assertEqual(r.status_code, 404)
+
+
+class TestApiValidateToken(ApiTokenMixin,
+                           BaseTestApiValidateToken,
+                           BaseTestCase):
+    user_model = User
+    radius_token_model = RadiusToken
+
+    def _get_url(self):
+        return reverse('freeradius:validate_auth_token',
+                       args=[self.default_org.slug])
+
+    def get_user(self):
+        user = self.user_model.objects.create_user(
+            username='test_name',
+            password='test_password',
+        )
```

Comment:
since `get_user` is already defined in `BaseTestApiValidateToken`, you should be able to avoid redefining this call to create the user by calling:
```python
user = super().get_user()
```

Please try this and let me know.

Reply:
@nemesisdesign it throwing this error:
```
UNIQUE constraint failed: openwisp_users_organizationuser.user_id, openwisp_users_organizationuser.organization_id
```

Reply:
ok